modules/site-health/admin-site-health.php

```diff
@@ -163,6 +168,13 @@ public function info_options( $debug_info ) {
 			}
 		}
 
+		$post_no_lang = $this->get_post_ids_without_lang();
+
+		if ( ! empty( $post_no_lang ) ) {
+			$fields['post-no-lang']['label'] = __( 'Posts without language', 'polylang' );
+			$fields['post-no-lang']['value'] = $this->format_array( $post_no_lang );
+		}
+
```

Comment:
The information is displayed in the Polylang options section. I would prefer that we create a new section (displayed only if there are posts without language).

Reply:
What do you think about creating a "Polylang Warning" section with the Post without ID inside and maybe other informations in the future ?

Reply:
Why not! Keep in mind that if this section is empty it mustn't be displayed. So if all its subsections are empty the section musn't be displayed.